public/docs/getting_started/console.md

```diff
@@ -1,36 +1,137 @@
-# Using the console
+# Using Truffle Develop and the console
 
-Sometimes it's nice to work with your contracts interactively for testing and debugging purposes, or for executing transactions by hand. Truffle provides you an easy way to do this via an interactive console, with your contracts available and ready to use.
+Sometimes it's nice to work with your contracts interactively for testing and debugging purposes, or for executing transactions by hand. Truffle provides you two easy ways to do this via an interactive console, with your contracts available and ready to use.
 
-## Command
+* **Truffle console**: A basic interactive console
+* **Truffle Develop**: An interactive console that also spawns a development blockchain
 
-To fire up the console, simply run:
+## Why two different consoles?
 
-```none
-$ truffle console
-```
+Having two different consoles allows you to choose the best tool for your needs.
+
+Reasons to use Truffle console:
+
+* You have a client you're already using, such as [Ganache](/docs/ganache/using) or geth
+* You want to migrate to a testnet (or the main Ethereum network)
+* You want to use a specific mnemonic or account list
+
+Reasons to use Truffle Develop:
+
+* You are testing your project with no intention of immediately deploying
+* You don't need to work with specific accounts
+* You don't need or want a separate blockchain running
```

Comment:
You still have a separate blockchain when using `truffle develop`, but you just don't have to manage it. I'd change it to: 

> You don't want to install and manage a separate blockchain client

Or something like that.